src/st/no3/377D4.c

```diff
@@ -327,8 +327,47 @@ void LoadObjLayout(s32 objLayoutId) {
 }
 #endif
 
+<<<<<<< HEAD
+#ifndef NON_MATCHING // TODO: try after resolving aspsx
+    INCLUDE_ASM("asm/st/no3/nonmatchings/377D4",
+                func_801C3E10); // https://decomp.me/scratch/cJ3CF
```

Comment:
Unfortunately we have to put the comments a line above `INCLUDE_ASM`, otherwise clang-format will make the instruction two lines and splat will fail to generate assembly code from that function